AmericanMedicalAssociation/ama-style-guide-2
# EWL-8304: Styling for Article Stub List embedded into article body text.

## Ticket(s)
**Do not** submit a Pull Request without a relevant JIRA ticket or Github issue! If you are creating a new pattern or feature, create an issue describing the need for this feature in Github **first** and then link to it below.

**Github Issue**
- [Issue XXX: Issue Title](https://github.com/AmericanMedicalAssociation/AMA-style-guide/issues/XXX)

**Jira Ticket**
- [EWL-8304: Allow Article Stub List As Inline to be embedded into article body text](https://issues.ama-assn.org/browse/EWL-8304)

## Description
Adjust styling of Article Stub grouping header when used as an embedded inline block.


## To Test
- Checkout and follow instructions on main PR -> [https://github.com/AmericanMedicalAssociation/ama-d8/pull/2339](https://github.com/AmericanMedicalAssociation/ama-d8/pull/2339)
- Link styleguides if needed
- Confirm styling matches the following Zeplins:

[Desktop](https://app.zeplin.io/project/59dd1f9c36a8b21d569bb9e8/screen/5f0632b0078c595cac7d27b9)

[Tablet](https://app.zeplin.io/project/59dd1f9c36a8b21d569bb9e8/screen/5dfd3469d852e81aeeba1419)

[Mobile](https://app.zeplin.io/project/59dd1f9c36a8b21d569bb9e8/screen/5e6959d61df0781127ed6e12)

## Visual Regressions

_Please provide the pull request ID below. This will then reference the automated VRT report after it has been generated._

A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/[PULL_REQUEST_ID]/html_report/index.html).

**Before proceeding:** Run visual regression tests locally to ensure new work does not introduce new bugs.

_The visual regression tests always compare against the production version of the style guide. Changes and new work will show up as test failures. Please describe your work so that the report can be reviewed by the team._

If you are creating or updating a pattern be sure you have created or updated the [scenario in `backstop.json`](ama-style-guide-2/styleguide/backstop.json).

_After changes in the report are documented and demonstrate your desired changes, please mark the pull request as `ready for review`.

For more information on visual regression testing review the [How do I run tests?](https://issues.ama-assn.org:8446/confluence/pages/viewpage.action?pageId=23298568) document in Confluence.


## Relevant Screenshots/GIFs
Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.


## Remaining Tasks
Things relevant to here may be updating [The Glossary](https://issues.ama-assn.org:8446/confluence/display/DTD/Glossary?src=contextnavpagetreemode) with the latest naming changes or review of the relative [D8](https://github.com/AmericanMedicalAssociation/ama-d8) work.


## Additional Notes

This is a styling PR for the following: [https://github.com/AmericanMedicalAssociation/ama-d8/pull/2339](https://github.com/AmericanMedicalAssociation/ama-d8/pull/2339)

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
